config/application.yml.default

```diff
@@ -219,7 +219,7 @@ pinpoint_voice_pool_size: 5
 piv_cac_service_url: https://localhost:8443/
 piv_cac_service_timeout: 5.0
 piv_cac_verify_token_url: https://localhost:8443/
-platform_authentication_enabled: true
+platform_auth_set_up_enabled: true
```

Comment:
Should this be `false` ?

```suggestion
platform_auth_set_up_enabled: false
```

Reply:
I wanted it to be enabled by default so that we have more control of when we overwrite it on deploy. That way after the code gets fully deployed we can update the config to turn it off in prod/leave it on in lower env.

Reply:
Hmm, I see. Personally I think most environments should behave like production (not production as an exception) and that the default configuration value should follow from this.

Maybe not a blocker then if you feel strongly enough about it. But we'd want to make sure this configuration would be in place in production before the deploy which includes these changes.